### PR TITLE
Change "Style" to "Line style" in properties panel

### PR DESCRIPTION
**Pull request by BrownianNotion**
Resolves: #13562. Resubmission of #19241 

TODO:
- [x] Don't apply change for slur/tie views

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

**Comment by cbjeukendrup** [comment]
But this is exactly the same as #19241 and thus [this comment](https://github.com/musescore/MuseScore/issues/13562#issuecomment-1710891623) still applies, right?

**Comment by BrownianNotion** [comment]
> But this is exactly the same as #19241 and thus [this comment](https://github.com/musescore/MuseScore/issues/13562#issuecomment-1710891623) still isn't fixed yet, right?

Sorry, missed that 😅 I'll change it

**Comment by BrownianNotion** [comment]
> But this is exactly the same as #19241 and thus [this comment](https://github.com/musescore/MuseScore/issues/13562#issuecomment-1710891623) still isn't fixed yet, right?

Fixed. New property is named `propertyPanelTitle`, but let me know if there's a more canonical name I could use.

File: src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineStyleSection.qml [hunk 1]
```diff
@@ -70,7 +72,7 @@ Column {
 
     FlatRadioButtonGroupPropertyView {
         id: styleSection
-        titleText: qsTrc("inspector", "Style")
+        titleText: qsTrc("inspector", root.propertyPanelTitle)
```
Comment (cbjeukendrup) [inline src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineStyleSection.qml:75]: Unfortunately, this way the translation system can't detect the string (it's not very smart). So, you should do:
```qml
property string propertyPanelTitle: qsTrc("inspector", "Line style")

…

titleText: root.propertyPanelTitle
```

Also, I propose to rename the property to something slightly more specific. What about `lineStyleTitleText`?

Comment (BrownianNotion) [reply src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineStyleSection.qml:75]: Thanks for the catch! `lineStyleTitleText` sounds good, will make the changes

**Comment by BrownianNotion** [comment]
@cbjeukendrup is this good to be merged? Let me know if there's anything else blocking

**Review by cbjeukendrup — APPROVED** [review]
Looks good to me, thanks!

**Comment by zacjansheski** [comment]
Tested on MacOS 14, Windows 11, Ubuntu 22.04.3. Approved
https://github.com/musescore/MuseScore/issues/13562 FIXED

**Pull request merged**